src/transforms.jl

```diff
@@ -221,4 +221,5 @@ include("transforms/quantile.jl")
 include("transforms/functional.jl")
 include("transforms/eigenanalysis.jl")
 include("transforms/sequential.jl")
-include("transforms/parallel.jl")
\ No newline at end of file
+include("transforms/parallel.jl")
+include("transforms/filter.jl")
```

Comment:
Similarly, please move this to after rename.jl. The last transforms should be sequential and parallel because they are meta-transforms.